lib/brain.js

```diff
@@ -605,7 +605,8 @@ Brain.prototype._connectedBrowser = function _connectedBrowser () {
     coins: this.trader.coins,
     twoWayMode: this.twoWayMode(),
     wifiList: wifiList,
-    rates
+    rates,
+    terms: this.trader.terms
```

Comment:
Will this update if the operator changes the terms screen on the backend?

Reply:
Probably need to do this in the `sendRates()` function. Since it's becoming more general than just sending rates, it should be renamed.

Reply:
cool, ok